From 3d5c92e221f0db991b5ca4f6d3765262e1616315 Mon Sep 17 00:00:00 2001
From: Viacheslav Login <vlogin@nvidia.com>
Date: Thu, 26 Sep 2024 10:28:54 +0400
Subject: [PATCH 28/33] [CI] Header check: upgrade version to NGCI v5.0.3-82

We're requested to upgrade the header checker tool (Copyright check tool) to latest version.

Issue: HPCINFRA-2612
Signed-off-by: Viacheslav Login <vlogin@nvidia.com>
---
 .ci/matrix_job.yaml                            | 8 ++++++--
 contrib/jenkins_tests/copyright-check-map.yaml | 2 +-
 2 files changed, 7 insertions(+), 3 deletions(-)

diff --git a/.ci/matrix_job.yaml b/.ci/matrix_job.yaml
index 5d0e771..20f11e8 100644
--- a/.ci/matrix_job.yaml
+++ b/.ci/matrix_job.yaml
@@ -14,6 +14,9 @@ kubernetes:
   limits: '{memory: 8Gi, cpu: 7000m}'
   requests: '{memory: 8Gi, cpu: 7000m}'
 
+credentials:
+  - {credentialsId: 'mellanox_github_credentials', usernameVariable: 'MELLANOX_GH_USER', passwordVariable: 'MELLANOX_GH_TOKEN'}
+
 volumes:
   - {mountPath: /hpc/local/bin, hostPath: /hpc/local/bin}
   - {mountPath: /hpc/local/oss, hostPath: /hpc/local/oss}
@@ -34,7 +37,7 @@ runs_on_dockers:
   - {name: 'fc31-mofed-x86_64',     url: 'harbor.mellanox.com/swx-infra/x86_64/fedora31/builder:mofed-5.1-1.0.7.0', category: 'base', arch: 'x86_64'}
   - {name: 'toolbox', url: 'harbor.mellanox.com/hpcx/x86_64/rhel8.3/builder:inbox', category: 'tool', arch: 'x86_64'}
   - {name: 'blackduck', url: 'harbor.mellanox.com/toolbox/ngci-centos:7.9.2009.2', category: 'tool', arch: 'x86_64'}
-  - {name: 'header-check', url: 'harbor.mellanox.com/toolbox/header_check:0.0.15', category: 'tool', arch: 'x86_64', tag: '0.0.15'}
+  - {name: 'header-check', url: 'harbor.mellanox.com/toolbox/header_check:0.0.51', category: 'tool', arch: 'x86_64', tag: '0.0.51'}
 
 runs_on_agents:
   - {nodeLabel: 'beni09', category: 'base'}
@@ -69,8 +72,9 @@ steps:
 
   - name: Copyrights
     enable: ${do_copyrights}
+    credentialsId: 'mellanox_github_credentials'
     run: |
-      env WORKSPACE=$PWD ./contrib/jenkins_tests/copyrights.sh
+      env WORKSPACE=$PWD GITHUB_TOKEN=$MELLANOX_GH_TOKEN ./contrib/jenkins_tests/copyrights.sh
     containerSelector:
       - "{name: 'header-check', category: 'tool', variant: 1}"
     agentSelector:
diff --git a/contrib/jenkins_tests/copyright-check-map.yaml b/contrib/jenkins_tests/copyright-check-map.yaml
index dde6b37..93eb65c 100644
--- a/contrib/jenkins_tests/copyright-check-map.yaml
+++ b/contrib/jenkins_tests/copyright-check-map.yaml
@@ -38,7 +38,7 @@ general:
         - ".*avner-test-orig"
         - ".*\\.m4"
 
-nbu-open-source:
+dual_gpl_2_bsd_3:
     include:
         - ".*"
     exclude:
-- 
2.43.0

